doc/examples/anim12/anim12.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Blending the NASA day and night views from the Blue and Black Marble mosaic
+# images using a day-night mask that goes through a full 24-hour cycle.  In addition,
+# we adjust the colors using the intensities derived from the slopes of the earth
+# relief grid.  We spin around at 24 frames per second where each frame advances by 1 minute.
+# Because we are not plotting anything, just manipulating images via grids, we must instead
+# create the frames and the final movie via the batch module.
+#
+# DEM:    @earth_relief_02m.grd
+# Images: @BlueMarble_02m.tif @BlackMarble_02m.tif
+# 
+# The finished movie is available in our YouTube channel as well:
+# https://youtu.be/X8TojLs0NYk
+# The movie took ~4 minutes to render on a 24-core MacPro 2013.
+# To make a UHD movie, comment/uncomment the SIZE, INC and FILT settings
+# SIZE=HD
+# INC=11.25
+# FILT=21
+SIZE=UHD
+INC=5.625
+FILT=10.5
+# 1. Create preflight script to build data files needed in the loop
+cat << EOF > pre.sh
+gmt begin
+  # Set view time for June 20-21, 2020 with steps of 1 minute
+  gmt math -T2020-06-20T/2020-06-21T/1 -o0 --TIME_UNIT=m  T = times.txt
+  # Create a ${SIZE} DEM at ${INC}x${INC} arc minutes to yield ${SIZE} dimensions
+  gmt grdfilter @earth_relief_04m -Fg${FILT} -I${INC}m -D1 -r -Gtopo_${SIZE}.grd
+  # Create an intensity grid based on the ${SIZE} DEM so we can see structures in the oceans
+  gmt grdgradient topo_${SIZE}.grd -Nt0.5 -A45 -Gintens_${SIZE}.grd
+gmt end
+EOF
+# 2. Set up main script
+cat << EOF > main.sh
+gmt begin
+  # Let HSV minimum value go to zero and faint map border
+  gmt set COLOR_HSV_MIN_V 0 MAP_FRAME_PEN=faint
+  # Make global grid with a smooth 2-degree day/night transition for this time.
+  gmt grdmath -Rd -I${INC}m -r \$(gmt solar -C -o0:1 -I+d\${BATCH_COL0}) 2 DAYNIGHT = daynight_${SIZE}.grd
+  # Blend the Blue and Black Marble geotiffs using the weights, so that when w is 1
+  # we get the Blue marble, and then adjust colors based on the intensity.
+  gmt grdmix @BlueMarble_${SIZE}.tif @BlackMarble_${SIZE}.tif -Wdaynight_${SIZE}.grd -Iintens_${SIZE}.grd -G\${BATCH_NAME}.png -Ve
```

Comment:
SIZE="UHD". Do we have the @BlueMarble_UHD.tif file?
```
gmt grdmix @BlueMarble_${SIZE}.tif @BlackMarble_${SIZE}.tif -Wdaynight_${SIZE}.grd -Iintens_${SIZE}.grd -G\${BATCH_NAME}.png -Ve
```

Reply:
Soon, so let's not merge until the files are ready.  Things are tied up [here](https://github.com/GenericMappingTools/gmtserver-admin/issues/37).

Reply:
I thought all the marble tiff files should have names like BlueMarble_01m.tif.

Reply:
Yes, but two problems:

1. For HD, that would be BlueMarble_11.25m.tiff, which seems annoying (?)
2. We have no way to downsample (filter) an image, e.g., grdfilter BlueMarble_06m.tif -Fg20 -D1 -GBlueMarble_HD.tif

The latter is solely due to @joa-quim being lazy.  Or, we stick in a **gdal_convert** call and do the downsampling that way.